### PR TITLE
Added missing profile links

### LINKED ISSUE #89: Added missing speaker links
Added:
+ Chan Cheow Hoe - https://www.facebook.com/chancheowhoe
+ Frank Karlitschek -  https://www.facebook.com/karlitschek
+ Vlado Koljibabic - https://www.facebook.com/vlado.koljibabic
+ Michael Christen - https://github.com/Orbiter
+ Philip Paeps - https://github.com/ppaeps
+ Simon Riggs - https://github.com/simonat2ndQuadrant
+ Franklin Weng -  https://github.com/fweng322
+ Melvin Zhang - https://www.facebook.com/melvinzhang
+ Lorenz Gerber - https://github.com/lorenzgerber

### PR DESCRIPTION
**Pull request by samyukthacodes**
Fixes #89. Please check the reference issue for details. The preview is available at https://samyukthacodes.github.io/2017.fossasia.org/index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -738,6 +746,7 @@ <h1>Featured Speakers</h1>
 											<a target="default" href="http://cosmiqo.com/"><i class="icon fa fa-external-link"></i></a>
                       <a target="default" href="https://github.com/melvinzhang"><i class="icon fa fa-github"></i></a>
 											<a target="default" href="https://twitter.com/melvinzhangzy"><i class="icon social_twitter"></i></a>
+											<a target="default" href="https://www.facebook.com/melvinzhang?hc_ref=ARQXjNouqW8ylbUiOyAsWzT1c1FcJuAC8pWvXITzr1gOXUKU0OFbjDG5pDgMzi53EWM"><i class="icon social_facebook"></i></a>
```
Comment (anshumanv) [inline index.html]: Replace - https://www.facebook.com/melvinzhang

File: index.html [hunk 2]
```diff
@@ -504,7 +508,8 @@ <h2>Vlado Koljibabic</h2>
 											<div class="social-links vertical-align">
 												<a target="default" href="https://www.linkedin.com/in/terekhov"><i class="icon social_linkedin"></i></a>
                                                                                                 <a target="default" href="https://github.com/andrey-terekhov"><i class="icon fa fa-github"></i></a>
-                                                                                                <a target="default" href="https://twitter.com/ddtru"><i class="icon social_twitter"></i></a>
+																								<a target="default" href="https://twitter.com/ddtru"><i class="icon social_twitter"></i></a>
```
Comment (anshumanv) [inline index.html]: Indent

**Review by anshumanv — CHANGES_REQUESTED** [review]
Check comments.

**Comment by samyukthacodes** [comment]
@anshumanv Hi. I made the edits you asked for. Could you please check them and see if it's alright?

File: index.html [hunk 3]
```diff
@@ -503,8 +507,9 @@ <h2>Vlado Koljibabic</h2>
 										<div class="hover-state text-center preserve3d">
 											<div class="social-links vertical-align">
 												<a target="default" href="https://www.linkedin.com/in/terekhov"><i class="icon social_linkedin"></i></a>
-                                                                                                <a target="default" href="https://github.com/andrey-terekhov"><i class="icon fa fa-github"></i></a>
-                                                                                                <a target="default" href="https://twitter.com/ddtru"><i class="icon social_twitter"></i></a>
+                                                <a target="default" href="https://github.com/andrey-terekhov"><i class="icon fa fa-github"></i></a>
```
Comment (anshumanv) [inline index.html]: Last indent, thanks for incorporating changes quickly. :smile:

**Comment by NaviRocker** [comment]
Please remove unwanted lines from the Code

**Comment by samyukthacodes** [comment]
@anshumanv I've fixed it. 
@naveentherocker Which unwanted lines should i fix?

File: index.html [hunk 4]
```diff
@@ -408,8 +409,8 @@ <h2>Chan Cheow Hoe</h2>
 											<div class="social-links vertical-align">
 												<a target="default" href="https://cloud.google.com/"><i class="icon fa fa-external-link"></i></a>
 												<a target="default" href="https://www.linkedin.com/in/oyvindr/"><i class="icon social_linkedin"></i></a>
-                                                                                                <a target="default" href="https://github.com/shimanokaze"><i class="icon fa fa-github"></i></a>
-                                                                                                <a target="default" href="https://github.com/shimanokaze"><i class="icon fa fa-github"></i></a>
+												<a target="default" href="https://github.com/shimanokaze"><i class="icon fa fa-github"></i></a>
+												<a target="default" href="https://github.com/shimanokaze"><i class="icon fa fa-github"></i></a>
```
Comment (ParthS007) [inline index.html]: @samyukthacodes Please add a link only once, Remove one GitHub link

**Review by ParthS007 — CHANGES_REQUESTED** [review]

File: index.html [hunk 5]
```diff
@@ -408,8 +409,8 @@ <h2>Chan Cheow Hoe</h2>
 											<div class="social-links vertical-align">
 												<a target="default" href="https://cloud.google.com/"><i class="icon fa fa-external-link"></i></a>
 												<a target="default" href="https://www.linkedin.com/in/oyvindr/"><i class="icon social_linkedin"></i></a>
-                                                                                                <a target="default" href="https://github.com/shimanokaze"><i class="icon fa fa-github"></i></a>
-                                                                                                <a target="default" href="https://github.com/shimanokaze"><i class="icon fa fa-github"></i></a>
+												<a target="default" href="https://github.com/shimanokaze"><i class="icon fa fa-github"></i></a>
```
Comment (isuruAb) [inline index.html:412]: please remove repeated lines

**Comment by samyukthacodes** [comment]
@ParthS007 @isuruAb Those changes were not made by me. They were already there. I thought of fixing them in a separate pull request. If you want I can fix them as well.

**Comment by samyukthacodes** [comment]
@ParthS007 @isuruAb I've made the changes you requested. Hope everything is alright now.

**Comment by samyukthacodes** [comment]
@isuruAb @ParthS007  @anshumanv  I've only two days remaining for this task which is a part of google code-in. If everything is fine could you please approve and merge this PR?

**Review by ParthS007 — APPROVED** [review]

**Comment by yashLadha** [comment]
Please resolve the merge conflicts.

**Comment by samyukthacodes** [comment]
@yashLadha Resolved

**Review by the-codingninja — APPROVED** [review]

**Review by saurabhjn76 — APPROVED** [review]

**Review by NaviRocker — APPROVED** [review]
LGTM

**Review by anshumanv — APPROVED** [review]

**Pull request merged**